app/models/petition.rb

```diff
@@ -335,6 +335,10 @@ def untagged_in_moderation
       untagged.in_moderation
     end
 
+    def recently_opened_or_signed(since=1.week.ago)
+      open_state.where(open_at: since..Time.current).or(where(last_signed_at: since..Time.current)).by_most_recent
+    end
```

Comment:
I would probably set these to be the beginning of the day and not dependent on the time of day when the method is called.